docs/integration/xbee-radio-integration.md

```diff
@@ -0,0 +1,152 @@
+# System Reference XBee Com Component Integration
+
+## Table of Contents
+
+* [1. Initializing XBee Radios](#1-ensure-xbee-devices-are-working-together)
+* [2. Topology Changes for UART Driver](#2-change-the-topology-of-the-com-components-to-include-the-linux-uart-driver)
+* [3. Instance Changes for Com Component](#3-change-the-instances-of-the-com-components-to-use-xbee-radio-com)
+* [4. Rerun build](#4-rerun-build)
+* [5. Running F Prime with UART Devices](#5-running-f-prime-with-uart-devices)
+    * [F Prime GDS](#f-prime-gds)
+    * [F Prime Binary](#f-prime-binary)
+
+## 1. Initializing XBee Radios
+
+We recommend you to test your XBee radios before implementing them into the F Prime ecosystem
+
+1. Download XCTU from [Digi](https://www.digi.com/products/embedded-systems/digi-xbee/digi-xbee-tools/xctu)
+2. Follow the steps to download the software and install it.
+3. Plug in XBee into device and do a default search for your XBee devices
+4. Open serial console and troubleshoot if they are not able to communicate with eachother
+
+[Link](https://www.digi.com/resources/documentation/digidocs/PDFs/90001458-13.pdf) to Digi documentation for XCTU.
+
+## 2. Topology Changes for UART Driver
+
+Find the following code snippets and uncomment them in the `SystemReference/Top/topology.fpp` file. 
+
+```
+Line: 111 * Subject to change * 
+#      rateGroup1Comp.RateGroupMemberOut[4] -> radio.run
+```
+```
+Line: 145-148 * Subject to change *
+#    connections Radio {
+#      radio.allocate -> comBufferManager.bufferGetCallee
+#      radio.deallocate -> comBufferManager.bufferSendIn
+#    }
+```
+
```

Comment:
I think we should avoid calling out specific lines, maybe we can add a comment to the section that needs to be uncommented in order to enable the driver?